app/src/main/java/com/dcrandroid/fragments/OverviewFragment.kt

```diff
@@ -391,15 +501,17 @@ class OverviewFragment : BaseFragment(), ViewTreeObserver.OnScrollChangedListene
         val totalBalanceCoin = Dcrlibwallet.amountCoin(totalBalanceAtom)
         if (isAdded) {
             val formattedUSD = HtmlCompat.fromHtml(
-                getString(
-                    R.string.usd_symbol_format,
-                    CurrencyUtil.dcrToFormattedUSD(exchangeDecimal, totalBalanceCoin, 2)
-                ), 0
+                    getString(
+                            R.string.usd_symbol_format,
+                            CurrencyUtil.dcrToFormattedUSD(exchangeDecimal, totalBalanceCoin, 2)
+                    ), 0
             )
 
             GlobalScope.launch(Dispatchers.Main) {
-                usdBalanceTextView.text = formattedUSD
-                usdBalanceTextView.show()
+                if (!isBalanceHidden) {
```

Comment:
We can use `MainScope().launch { }` to replace `GlobalScope`

Reply:
what's the advantage of `MainScope().launch { }` in this case?